type or namespace Xunit not found

I'm running a fresh install of dotnet sdk 2.1.301 on ubuntu 18.04
I ran "dotnet new xunit" and got the test files described, however I am getting the error CS0246: The type or namespace name 'Xunit' could not be found (are you missing a using directive or an assembly reference?)

Any ideas?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f9b54562-9072-a864-be3d-44ed7965c798
* Version Independent ID: a69ec985-ed05-90dd-a686-dc956b0cf5df
* Content: [Organizing and testing projects with the .NET Core command line](https://docs.microsoft.com/en-us/dotnet/core/tutorials/testing-with-cli)
* Content Source: [docs/core/tutorials/testing-with-cli.md](https://github.com/dotnet/docs/blob/master/docs/core/tutorials/testing-with-cli.md)
* Product: **dotnet-core**
* GitHub Login: @cartermp
* Microsoft Alias: **mairaw**

--- a/docs/core/tutorials/testing-with-cli.md
+++ b/docs/core/tutorials/testing-with-cli.md
@@ -97,19 +97,19 @@ Optional exercise: You can add a new pet type, such as a `Bird`, by extending th
 
 The `NewTypes` project is in place, and you've organized it by keeping the pets-related types in a folder. Next, create your test project and start writing tests with the [xUnit](https://xunit.github.io/) test framework. Unit testing allows you to automatically check the behavior of your pet types to confirm that they're operating properly.
 
-Create a *test* folder with a *NewTypesTests* folder within it. At a command prompt from the *NewTypesTests* folder, execute `dotnet new xunit`. This produces two files: *NewTypesTests.csproj* and *UnitTest1.cs*.
+Navigate back to the *src* folder and create a *test* folder with a *NewTypesTests* folder within it. At a command prompt from the *NewTypesTests* folder, execute `dotnet new xunit`. This produces two files: *NewTypesTests.csproj* and *UnitTest1.cs*.
 
 The test project cannot currently test the types in `NewTypes` and requires a project reference to the `NewTypes` project. To add a project reference, use the [`dotnet add reference`](../tools/dotnet-add-reference.md) command:
 
 ```
-dotnet add reference ../../src/NewTypes/NewTypes.csproj
+dotnet add reference ../../NewTypes/NewTypes.csproj
 ```
 
-You also have the option of manually adding the project reference by adding an `<ItemGroup>` node to the *NewTypesTests.csproj* file:
+Or, you also have the option of manually adding the project reference by adding an `<ItemGroup>` node to the *NewTypesTests.csproj* file:
 
 ```xml
 <ItemGroup>
-  <ProjectReference Include="../../src/NewTypes/NewTypes.csproj" />
+  <ProjectReference Include="../../NewTypes/NewTypes.csproj" />
 </ItemGroup>
 ```
 

